docker-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
     container_name: wordle-pg
     restart: always
     image: postgres:14.3
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
Remove this.. unless your docker container cannot start without it.

Reply:
alright